Scripts/datahandling/traversaldata.py

```diff
@@ -0,0 +1,93 @@
+from pathlib import Path
+import parameters.assignment as param
+import numpy
+
+def transform_traversal_data(result_path: Path, zones: list):
+    """Processes freight model specific traversal files containing
+    information on amount of transported tons between gate pair.
+    Processed traversal file contents are aggregated as auxiliary matrix.
+
+    Parameters
+    ----------
+    result_path : Path
+        result path where traversal files are located
+    zones : list
+        zones in network
+
+    Returns
+    ----------
+    numpy matrix
+        Aggregated auxiliary demand of freight modes
+    """
+    aux_tons = numpy.zeros([len(zones), len(zones)], dtype=numpy.float32)
+    for ass_class in param.freight_modes:
+        file = result_path / f"{ass_class}.txt"
+        aux_tons += read_traversal_file(file, numpy.array(zones))
```

Comment:
I was also thinking of passing the zeros matrix to the next func and performing addition there but this seems better to me.